docs/src/pinn/parm_estim.md

```diff
@@ -0,0 +1,67 @@
+# Optimising Parameters of a Lorenz System
+ Consider a Lorenz System ,
+! [lorenzSystem](https://raw.githubusercontent.com/ashutosh-b-b/github-doc-images/master/lorenz_system.png) with Physics-Informed Neural Networks.
+Now we would consider the case when we know σ = 10 and we want to optimise β  and ρ.
+We start by defining the the problem,
+```julia
```

Comment:
Add all used dependencies so that the example can be easy copied-past like in other examples

Reply:
Bump